dlk/python/dlk/core/optimizer.py

```diff
@@ -19,13 +19,66 @@
 
 from core.graph import Graph
 from core.graph_pattern_matching import get_nodes_in_branch, sort_graph
-from core.operators import Constant, Operator
+from core.operators import Constant, Operator, Conv
 from core.data_types import Uint32, QUANTIZED_NOT_PACKED
-from typing import cast
+from typing import cast, List, Any
 from collections import defaultdict
 from modules.packer import Packer
 
 
+def transpose_kernels(kernel_data: np.ndarray,
```

Comment:
Please add `_` prefix because it is not public (global) function, and please write comment of what is this function.

Reply:
I fixed it.